core/services/ardupilot_manager/mavlink_proxy/Endpoint.py

```diff
@@ -1,15 +1,15 @@
 from dataclasses import dataclass
-from enum import IntEnum
+from enum import Enum
 from typing import Dict, Optional, Union
 
 
-class EndpointType(IntEnum):
-    Undefined = 0
-    UDPServer = 1
-    UDPClient = 2
-    TCP = 3
-    Serial = 4
-    File = 5
+class EndpointType(str, Enum):
+    Undefined = "undefined"
+    UDPServer = "udpin"
+    UDPClient = "udpout"
+    TCP = "tcp"
+    Serial = "serial"
+    File = "file"
```

Comment:
EndPoint class should be updated to use the strings of EddpointType over connTypeMap

Reply:
You mean using udp instead of udpout?

Reply:
I'm saying that `connTypeMap` should not exist, since this enum will do the job.

Reply:
Oh, sure, this makes sense!